instrumentation/p6spy/src/main/java/brave/p6spy/TracingP6SpyOptions.java

```diff
@@ -1,32 +1,58 @@
 package brave.p6spy;
 
-import com.p6spy.engine.spy.P6SpyOptions;
+import com.p6spy.engine.logging.P6LogLoadableOptions;
+import com.p6spy.engine.logging.P6LogOptions;
+import com.p6spy.engine.spy.P6LoadableOptions;
 import com.p6spy.engine.spy.option.P6OptionsRepository;
+
+import javax.management.StandardMBean;
+import java.util.LinkedHashMap;
 import java.util.Map;
 
-final class TracingP6SpyOptions extends P6SpyOptions {
+final class TracingP6SpyOptions extends StandardMBean
+    implements P6LoadableOptions, TracingP6SpyOptionsMBean {
+
   static final String REMOTE_SERVICE_NAME = "remoteServiceName";
   static final String INCLUDE_PARAMETER_VALUES = "includeParameterValues";
+  private static final Map<String, String> defaults = new LinkedHashMap();
 
-  final P6OptionsRepository optionsRepository;
+  private final P6OptionsRepository optionsRepository;
+  private final P6LogLoadableOptions logLoadableOptions;
+
+  static {
+    defaults.put(INCLUDE_PARAMETER_VALUES, Boolean.FALSE.toString());
+  }
 
-  TracingP6SpyOptions(P6OptionsRepository optionsRepository) {
-    super(optionsRepository);
+  public TracingP6SpyOptions(P6OptionsRepository optionsRepository) {
```

Comment:
revert public here. note: please use the format on all files changed as I notice some missing or too many spaces around. if you look at the diff you can notice.